### PR TITLE
chore: add additional rules for _fieldExample

### PR DESCRIPTION
**Pull request by SSelinn**
Added additional rules to make _fieldExample not a warning but the default for internal, private, and protected_internal settings.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -253,3 +253,14 @@ dotnet_style_operator_placement_when_wrapping = beginning_of_line
 tab_width = 4
 indent_size = 4
 end_of_line = crlf
+
+# Use underscores for private fields
+dotnet_naming_symbols.private_fields.applicable_kinds           = field
+dotnet_naming_symbols.private_fields.applicable_accessibilities = internal, protected_internal, private 
+
+dotnet_naming_style.prefix_underscore.capitalization = camel_case
```
Comment (Tboefijn) [inline .editorconfig]:
```suggestion
dotnet_naming_style.prefix_underscore.capitalization  = camel_case
```

**Review by Tboefijn — CHANGES_REQUESTED** [review]
LGTM! You missed one space.

**Review by Tboefijn — APPROVED** [review]
LGTM!

**Review by QuakeEye — APPROVED** [review]
image

**Pull request merged**